doc/making-a-visualizer.md

```diff
@@ -62,7 +60,7 @@ you can set in the `params` property.
 -   **Differences:** `n = 20`, `levels = 5`. The parameters are refreshed in
     the in the `inhabit()` method.
```

Comment:
None of this comment has been addressed, so far as I can tell -- remember to unhide all comments and check through all the open conversations as you are revising.

Reply:
Good points. I've tried to explain the Differences visualizer's parameter refresh in a more meaningful way, and to make it clear that this is specific to Differences.

Reply:
Hmmm, I haven't looked at this at all because I just keep checking the "in the in the" and the outdated reference to `inhabit()` and they haven't changed. I'll take another look at the semantic/expository aspects when the mechanical ones are fixed.

Reply:
Ah, this explanation is very nice now, thank you! I just detected one point of possible confusion between "the user" and "you" (who are different people in this text) and so made a quick small PR amplifying that and making headings consistently nouns for data, verbs for methods, and "How to" for everything else. Resolving.